Makefile

```diff
@@ -144,7 +144,7 @@ pvs_studio:
 
 # call the Artistic Style pretty printer on all source files
 pretty:
-	astyle \
+	/Users/niels/Downloads/astyle \
```

Comment:
Probably don't want to include this change.  :)

Reply:
You're a life saver... Good this is still a draft...